tests/unit_tests/test_features.py

```diff
@@ -151,6 +163,23 @@ def test_strategy_variant_is_returned(test_feature_strategy_variants):
         "enabled": True,
         "name": "VarC",
         "payload": {"type": "string", "value": "Test 3"},
+        "is_feature_enabled": True,
+    }
+
+
+def test_feature_enabled_when_no_variants(test_feature_no_variants):
+    context = {
+        "userId": "122",
+        "appName": "test",
+        "environment": "prod",
+        "customField": "1",
+    }
+    variant = test_feature_no_variants.get_variant(context)
+
+    assert variant == {
+        "enabled": False,
+        "name": "disabled",
+        "is_feature_enabled": True,
```

Comment:
Why did we choose `is_feature_enabled` over `feature_enabled`?

Reply:
It's the name suggested in the [issue](https://github.com/Unleash/unleash-client-python/issues/284#issue-1973288429), and I noticed we also have a variable with this name in the [existing codebase](https://github.com/Unleash/unleash-client-python/pull/285/files#diff-6ca66179fe5f5e0a1ecaca00106886de49ea4860733d5009b83a1480c44aa43aR122).
If you prefer `feature_enabled` let me know, I can easily make the change 🙂

Reply:
Don't know that I would have picked this up but now that we're talking, I'm leaning towards "feature_enabled" rather